### PR TITLE
Developer UI: Send transaction from an app

### PR DESCRIPTION
**Pull request by mikheevm**
Review after: 
- https://github.com/gnosis/safe-apps-sdk/pull/134
- https://github.com/gnosis/safe-apps-sdk/pull/135
- https://github.com/gnosis/safe-apps-sdk/pull/136

This PR:
- Allows sending transaction from an app

### CODE REVIEW

File: packages/safe-apps-developer-ui/src/components/pages/safes/apps/AppIframe.tsx [hunk 1]
```diff
@@ -13,9 +20,16 @@ const SIframe = styled.iframe`
   height: 100%;
 `;
 
+type ProposedTxs = {
+  transactions: Transaction[];
+  requestId: RequestId;
+  params: SendTransactionParams;
```
Comment (germartinez) [inline packages/safe-apps-developer-ui/src/components/pages/safes/apps/AppIframe.tsx:26]: Are these params used somewhere?

File: packages/safe-apps-developer-ui/src/api/safeContracts.ts [hunk 2]
```diff
@@ -1,34 +1,54 @@
 import GnosisSafeSol from '@gnosis.pm/safe-contracts/build/contracts/GnosisSafe.json';
+import MultiSendSol from '@gnosis.pm/safe-contracts/build/contracts/MultiSend.json';
 import ProxyFactorySol from '@gnosis.pm/safe-contracts/build/contracts/GnosisSafeProxyFactory.json';
 import FallbackHandlerSol from '@gnosis.pm/safe-contracts/build/contracts/FallbackManager.json';
 import { JsonRpcSigner } from '@ethersproject/providers';
 import { Contract, ContractFactory } from 'ethers';
+import { GnosisSafe, GnosisSafeProxyFactory, MultiSend, MasterCopy, FallbackManager } from 'src/types/contracts';
```
[the remark this answers is not in the document]
Comment (mikheevm) [reply packages/safe-apps-developer-ui/src/api/safeContracts.ts:7]: It's generated via typeschain at postInstall: https://github.com/gnosis/safe-apps-sdk/pull/133/files#diff-9557b21303b92a38d382819196b4b08ba5410aeca1c8869da23f9f047cff3ea4R34

File: packages/safe-apps-developer-ui/src/api/transactions.ts [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+import { ethers } from 'ethers';
+import { getSafeContract, getMultiSendContract } from 'src/api/safeContracts';
+import { ProposedTx, CreateTransactionArgs } from 'src/types/transaction';
+import { getPreValidatedSignature } from './signatures';
+import { getSafeNonce } from './safe';
+
+const CALL = 0;
+const DELEGATE_CALL = 1;
+
+const approveTransactionHash = async (
```
[the remark this answers is not in the document]
Comment (mikheevm) [reply packages/safe-apps-developer-ui/src/api/transactions.ts:13]: I think I used copy-paste programming paradigm and accidentally left it there.

File: packages/safe-apps-developer-ui/src/components/pages/safes/apps/AppIframe.tsx (hunk 1 above)
Comment (mikheevm) [reply packages/safe-apps-developer-ui/src/components/pages/safes/apps/AppIframe.tsx:26]: now they're passed to the tx creation func, good catch!

**Review by germartinez — APPROVED** [review]

**Pull request merged**